sympy/physics/vector/frame.py

```diff
@@ -475,72 +475,40 @@ def dcm(self, otherframe):
         self._dcm_cache[otherframe] = outdcm
         otherframe._dcm_cache[self] = outdcm.T
         return outdcm
-
-    def orient(self, parent, rot_type, amounts, rot_order=''):
-        """Sets the orientation of this reference frame relative to another
-        (parent) reference frame.
-
-        Parameters
-        ==========
-
-        parent : ReferenceFrame
```

Comment:
Don't remove any public methods. This method should call all of the new methods.